component/block_cache/block_cache.go

```diff
@@ -1535,9 +1536,10 @@ func (bc *BlockCache) stageZeroBlock(handle *handlemap.Handle, tryCnt int) (stri
 
 	log.Debug("BlockCache::stageZeroBlock : Staging zero block for %v=>%v, try = %v", handle.ID, handle.Path, tryCnt)
 	err := bc.NextComponent().StageData(internal.StageDataOptions{
-		Name: handle.Path,
-		Data: bc.blockPool.zeroBlock.data[:],
-		Id:   id,
+		Name:   handle.Path,
+		Data:   bc.blockPool.zeroBlock.data[:],
+		Offset: 0,
```

Comment:
This can be removed as the default value is 0 and for staging a block offset should not be required.